packages/firefuel/test/src/firefuel_test.dart

```diff
@@ -16,4 +16,38 @@ void main() {
       expect(Firefuel.firestore, isA<FirebaseFirestore>());
     });
   });
+
+  group('#env', () {
+    test('should return an empty string when not initialized', () {
+      expect(Firefuel.env, isEmpty);
+    });
+
+    test('should return an empty string when not provided', () {
+      Firefuel.initialize(FakeFirebaseFirestore());
+
+      expect(Firefuel.env, isEmpty);
+    });
+
+    test('should return string starting with prefix when provided', () {
+      const env = 'test';
+
+      Firefuel.initialize(
+        FakeFirebaseFirestore(),
+        env: env,
+      );
+
+      expect(Firefuel.env.startsWith(env), isTrue);
```

Comment:
Use a more specific matcher. If this test fails it will give a better description of why it failed rather than "expected to be True"

```suggestion
      expect(Firefuel.env, startsWith(env));
```